packages/components/src/components/vertical-tabs/vertical-tab-headers/vertical-tab-headers.jsx

```diff
@@ -0,0 +1,23 @@
+import React                  from 'react';
+import VerticalTabHeader      from 'Components/vertical-tabs/vertical-tab-header';
```

Comment:
I think we have a "convention" of sorts to include `.jsx` in the imports. I'm not sure tho.

Reply:
That won't work cause we're importing the `index.js` here, which in turn imports the component's CSS. If we specifically import the component we'll lose the styling.